### PR TITLE
Add dropping syn packets when config file was updated to avoid down time

### PR DESCRIPTION
**Pull request by maxbr**
I had rewrited the script accordingly to https://engineeringblog.yelp.com/2015/04/true-zero-downtime-haproxy-reloads.html article to archive zero downtime when haproxy config changes.

### CODE REVIEW

File: files/haproxy_reload.sh [hunk 1]
```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```
Comment (ernestas-poskus) [inline files/haproxy_reload.sh]: mode from files/haproxy_reload.sh to templates/haproxy_reload.sh.j2

File: files/haproxy_reload.sh [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+
+function api_response {
+  curl -skL http://127.0.0.1:8080/v2/tasks -H "Accept: text/plain"
```
Comment (ernestas-poskus) [inline files/haproxy_reload.sh]: make 8080 port configurable, playbook has ```marathon_port``` variable

**Comment by maxbr** [comment]
updated

**Comment by ernestas-poskus** [comment]
Thank you for contribution

**Pull request merged**